sedaro/src/sedaro/results/simulation.py

```diff
@@ -45,12 +45,18 @@ def __repr__(self) -> str:
         return f'SedaroSimulationResult(branch={self.__branch}, status={self.status})'
 
     @classmethod
-    def get_scenario_latest(cls, api_key: str, scenario_id: int, host: str = 'https://api.sedaro.com'):
+    def get_scenario_latest(
+        cls,
+        api_key: str,
+        scenario_id: int,
+        host: str = 'https://api.sedaro.com',
+        streams: List[Tuple[str, ...]] = []
```

Comment:
I'd have to dig up the reference but are you familiar with the anti-pattern of initializing kwargs to lists, dicts, etc. (mutable types)?  It can introduce very tricky bugs.  Instead, please use the following

```py
def get_scenario_latest(..., streams=None):
  streams = streams or []
  ...
```